source/catalog.html

```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Пинк</title>
+  </head>
+  <body class="page__body">
+    <header class="page-header">
+      <a class="page-header__logo">
+        <img src="img/logo-cut.svg" width="74" height="23" alt="PINK logotype">
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list site-list">
+            <li class="site-list__item">
+              <a href="index.html">
+                Главная
+              </a>
+            </li>
+            <li class="site-list__item">
+              <a>
+                Фотографии
+              </a>
+            </li>
+            <li class="site-list__item">
+              <a href="form.html">
+                Конкурс
+              </a>
+            </li>
+            <li class="site-list__item">
+              <a href="https://htmlacademy.ru/intensive/adaptive">
+                HTML Academy
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="page-main">
+      <h1 class="visually-hidden">Каталог</h1>
+      <section class="catalog-page">
+        <div class="catalog-page__wrapper">
+          <header class="catalog-page__header">
+            <h2 class="visually-hidden">Фотографии<br>
+              пользователей</h2>
+          </header>
+          <div class="catalog-page__text-top">
+            <p>Взгляните на фотографии, которые выкладывают пользователи! Видите, как не хватает ваших?</p>
+          </div>
+          <ul class="catalog-page__list catalog-photo">
+            <li class="catalog-photo_item catalog-photo__item--best">
```

Comment:
```suggestion
            <li class="catalog-photo__item catalog-photo__item--best">
```
Во всех айтемах нужно подправить